src/Chainweb/Pact/PactService.hs

```diff
@@ -1607,3 +1659,57 @@ freeModuleLoadGasModel = modifiedGasModel
 --
 officialGasModel :: P.GasModel
 officialGasModel = freeModuleLoadGasModel
+
+-- -------------------------------------------------------------------------- --
```

Comment:
I might argue that we make a new module instead of worrying about formatting this over-large module, a Checkpointer sub-module would be great

Reply:
Agreed. Moving this into something like `PactService.Checkpointer` would make sense.

I had the code already moved into this sections to avoid intense scrolling during editing them. I only moved the code back in place to get nice diffs and make the review easier.

I propose that we do a follow up PR, that only moves the code but doesn't change any behavior.